bulbs/poll/api.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.routers import DefaultRouter
+
+from .viewsets import PollViewSet, AnswerViewSet
+
+router = DefaultRouter(trailing_slash=True)
+
+api_v1_router = DefaultRouter()
+api_v1_router.register(
```

Comment:
Combine also has an experimental precommit hook for Git that runs lint + tests. 

VIM has support for flake8 (I assume lots of editors do). Also created scripts/lint.

Flake8 is pretty popular and just is a wrapper around "pep8 and pyflakes" linters.